src/main/java/com/atlassian/bitbucket/jenkins/internal/model/BitbucketCommit.java

```diff
@@ -12,36 +13,27 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class BitbucketCommit {
 
-    private final long committerTimestamp;
-    private final String displayId;
-    private final String id;
-    private final String message;
+    public ArrayList<BitbucketValueProperty> values;
 
     @JsonCreator
-    public BitbucketCommit(
-            @JsonProperty("id") String id,
-            @JsonProperty("displayId") String displayId,
-            @JsonProperty("committerTimestamp") long committerTimestamp,
-            @JsonProperty("message") String message) {
-        this.id = requireNonNull(id, "id");
-        this.displayId = requireNonNull(displayId, "displayId");
-        this.message = message;
-        this.committerTimestamp = committerTimestamp;
+    public BitbucketCommit(@JsonProperty("values") ArrayList<BitbucketValueProperty> values) {
```

Comment:
The problem with this representation is if we ever need to fetch a single commit from an endpoint, not multiple, this data object no longer works (there will be no `values` array).